packages/@lwc/engine-dom/tsconfig.json

```diff
@@ -5,5 +5,5 @@
         "lib": ["dom", "es2021"]
     },
 
-    "include": ["src/"]
+    "include": ["src/", "./html.d.ts"]
```

Comment:
This `html.d.ts` file is not actually going to get included in the published npm package. You can use `npx pkgfiles` to confirm.

Since this is just a convenience for our users, I would simplify things and just put this directly into the `lwc` package. It doesn't strictly need to be in `@lwc/engine-dom` at all.

Reply:
> It doesn't strictly need to be in @lwc/engine-dom at all.

> It breaks with our convention of "The lwc package just re-exports other packages"

Pick one! 😛